### ISSUE TITLE
DB query error

### ISSUE THREAD

**Issue by Mmj94** [imr-framework/virtual-scanner#3]
UnboundLocalError: local variable 'conn' referenced before assignment
[UnboundLocalError_ local variable 'conn' referenced before assignment __ Werkzeug Debugger.pdf](https://github.com/imr-framework/Virtual-Scanner/files/3233899/UnboundLocalError_.local.variable.conn.referenced.before.assignment.__.Werkzeug.Debugger.pdf)

**Comment by sairamgeethanath** [comment]
The fix is to have the 'coms_server_flask.py' script run from the Virtual Scanner folder, this is an IDE (PyCharm) issue, the script automatically works fine from command line interface

**Comment by sravan953** [comment]
Running the script from the command line does not work either.
Also, there must be a simpler fix as opposed to changing working directory to work with PyCharm?

**Comment by sairamgeethanath** [comment]
This issue was fixed with the following lines for CLI if __name__ == '__main__':
    import os
    import sys

    script_path = os.path.abspath(__file__)
    SEARCH_PATH = script_path[:script_path.index('Virtual-Scanner') + len('Virtual-Scanner') + 1]
    sys.path.insert(0, SEARCH_PATH)

Please do provide screenshots or error messages when you run from CLI. In parallel, I am not aware of a bug fix for config edits in PyCharm, happy to receive any inputs

**Comment by sravan953** [comment]
Commit https://github.com/imr-framework/virtual-scanner/commit/bab61123f71fbee86229e0cefbdda76e70723a44 fixes this issue.

1. PyCharm working directory changes are no longer needed.
2. Code runs fine from command line too.
3. `subject.db` is only created in the expected directory and not in the user's home directory.

Please confirm from your end.

**Comment by sairamgeethanath** [comment]
Ran the code 3 times with no failure for the path related to subject.db.
Bug fixed.
